integrations/chroma/pyproject.toml

```
@@ -26,7 +26,7 @@ classifiers = [
 ]
 dependencies = [
   "haystack-ai",
-  "chromadb<0.4.20",  #  FIXME: investigate why filtering tests broke on 0.4.20
+  "chromadb>=0.5.0",
```

```suggestion
  "chromadb",
```

I would remove the pin to a specific version.